lighthouse-core/audits/works-offline.js

```diff
@@ -39,8 +39,16 @@ class WorksOffline extends Audit {
    * @return {!AuditResult}
    */
   static audit(artifacts) {
+    let debugString;
+    if (artifacts.URL.initialUrl !== artifacts.URL.finalUrl) {
```

Comment:
do you want to use `URL.equalWithExcludedFragments` here? I don't see how you could get into a redirect situation where it would matter, but as far as network requests are concerned, with or without the fragment would be the same URL.